### ISSUE TITLE
FATAL: Error, failed to find path bazel-out/osx-x86_64-opt-ST-b50b65861838/bin/external/upb/python/lib_message_binary.so 

### ISSUE THREAD

**Issue by brunoonovais** [Nuitka/Nuitka#1887]
## error ##

```
 % python -m nuitka --standalone --follow-imports gnmi_subscribe_xpath_list.py
Nuitka-Options:INFO: Used command line options: --standalone --follow-imports gnmi_subscribe_xpath_list.py
Nuitka-Options:INFO: Following all imports is the default for standalone mode and need not be specified.
Nuitka:WARNING: Using very slow fallback for ordered sets, please install 'orderedset' PyPI package for best Python compile time performance.
Nuitka:INFO: Starting Python compilation with Nuitka '1.1.6' on Python '3.9' commercial grade 'not installed'.
Nuitka-Plugins:WARNING: Use '--enable-plugin=numpy' for: Numpy may miss DLLs otherwise.
Nuitka-Plugins:INFO: multiprocessing: Injecting pre-module load code for module 'multiprocessing':   
Nuitka-Plugins:INFO: multiprocessing:     Monkey patching "multiprocessing" load environment.
Nuitka-Plugins:INFO: multiprocessing: Injecting post-module load code for module 'multiprocessing':
Nuitka-Plugins:INFO: multiprocessing:     Monkey patching "multiprocessing" for compiled methods.
Nuitka:INFO: Completed Python level compilation and optimization.                                    
Nuitka:INFO: Generating source code for C backend compiler.
Nuitka:INFO: Running data composer tool for optimal constant value handling.                                     
Nuitka:INFO: Running C compilation via Scons.
Nuitka-Scons:INFO: Backend C compiler: clang (clang).
Nuitka-Scons:INFO: Backend linking program with 243 files (no progress information available).
Nuitka-Scons:INFO: Running /usr/bin/clang -o /Users/bnovais/Dropbox/PycharmProjects/A/gnmi_subscribe_xpath_list.dist/gnmi_subscribe_xpath_list -flto --target=arm64-apple-macos10.9 -O3 -Wno-deprecated-declarations -headerpad_max_install_names @"./@link_input.txt -L/Library/Frameworks/Python.framework/Versions/3.9/lib -lm -lpython3.9 took 179.42 seconds
Nuitka-Scons:INFO: Compiled 1937 C files using ccache.
Nuitka-Scons:INFO: Cached C files (using ccache) with result 'cache miss': 242
Nuitka-Scons:INFO: Cached C files (using ccache) with result 'local_storage_miss': 242
Nuitka-Scons:INFO: Cached C files (using ccache) with result 'local_storage_read_miss': 484
Nuitka-Scons:INFO: Cached C files (using ccache) with result 'local_storage_write': 484
FATAL: Error, failed to find path bazel-out/osx-x86_64-opt-ST-b50b65861838/bin/external/upb/python/lib_message_binary.so (resolved DLL to bazel-out/osx-x86_64-opt-ST-b50b65861838/bin/external/upb/python/lib_message_binary.so) for /Users/bnovais/Dropbox/PycharmProjects/venv-3.10/lib/python3.9/site-packages/google/_upb/_message.abi3.so, please report the bug.
```

## nuitka version

```
python -m nuitka --version
1.1.6
Commercial: None
Python: 3.9.13 (v3.9.13:6de2ca5339, May 17 2022, 11:37:23) 
Flavor: Unknown
Executable: /Users/bnovais/Dropbox/PycharmProjects/venv-3.10/bin/python
OS: Darwin
Arch: arm64
```

## freeze

```
% python -m pip freeze
appnope==0.1.3
asttokens==2.0.5
backcall==0.2.0
bcrypt==4.0.0
certifi==2022.9.14
cffi==1.15.1
charset-normalizer==2.1.1
cryptography==38.0.1
decorator==5.1.1
dictdiffer==0.9.0
distlib==0.3.6
executing==0.9.1
filelock==3.8.0
future==0.18.2
grpcio==1.49.1
idna==3.4
ipython==8.4.0
ixnetwork-restpy==1.1.6
jedi==0.18.1
jsonrpclib==0.2.1
matplotlib-inline==0.1.3
netmiko==4.1.2
ntc-templates==3.1.0
Nuitka==1.1.6
numpy==1.23.3
paramiko==2.11.0
parso==0.8.3
pexpect==4.8.0
pickleshare==0.7.5
platformdirs==2.5.2
prettytable==3.4.1
prompt-toolkit==3.0.30
protobuf==4.21.6
ptyprocess==0.7.0
pure-eval==0.2.2
pycparser==2.21
Pygments==2.12.0
pygnmi==0.8.9
PyNaCl==1.5.0
pyserial==3.5
PyYAML==6.0
requests==2.28.1
scp==0.14.4
six==1.16.0
stack-data==0.3.0
tenacity==8.1.0
textfsm==1.1.2
traitlets==5.3.0
urllib3==1.26.12
virtualenv==20.16.5
virtualenv-tools3==2.0.5
wcwidth==0.2.5
websocket-client==1.4.1
```

- installed nuitka from pip

**Comment by kayhayen** [comment]
The `ccache` related output looks interesting, probably a new ccache version I have not seen before, but that is an aside. It seems you failed provide a minimal example.

**Comment by kayhayen** [comment]
Closing for lack of feedback.

**Comment by photuris** [comment]
I'm having basically the same issue. Anyone have any idea what might be happening?

```bash
Nuitka-Plugins:INFO: dll-files: Found 7 files DLLs from torch installation.
FATAL: Error, failed to find path bazel-out/osx-x86_64-opt-ST-7396233f5981/bin/external/upb/python/lib_message_binary.so (resolved DLL to bazel-out/osx-x86_64-opt-ST-7396233f5981/bin/external/upb/python/lib_message_binary.so) for /Users/joshua/Projects/de_id/.venv/lib/python3.11/site-packages/google/_upb/_message.abi3.so, please report the bug.
```

**Comment by kayhayen** [comment]
@photuris posting below closed issues is a sure-fire way to get ignored, please open a new issue.
